demos/prefsFramework/html/SimplifyPanelTemplate.html

```diff
@@ -2,9 +2,16 @@
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="simplify-choice" class="demo-prefsEditor-simplify" />
     <label for="simplify-choice">
-    	<span class="demo-prefsEditor-simplify-descr"></span>
-        <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
-            <span class="fl-prefsEditor-switch-inner"></span>
+    	<span class="demo-prefsEditor-simplify-descr fl-prefsEditor-panel-descr"></span>
```

Comment:
ESLINT only looks at the JS files. We'd probably need an HTML linter to catch this.